src/analysis/explorerHomeView.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+import {CredGraph} from "../core/credrank/credGraph";
+import {type TimestampMs} from "../util/timestamp";
+import {type Interval} from "../core/interval";
+import * as Grain from "../core/ledger/grain";
```

Comment:
nit / no change required: If I had written this, I might do something like:

```js
import * as Grain from "../core/ledger/grain";
import type {Grain as GrainT} from "../core/ledger/grain";
```

Reply:
Ah yes, I like that.